<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SporadicFrameEditDlg</class>
 <widget class="QDialog" name="SporadicFrameEditDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>666</width>
    <height>455</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Create Sporadic Frame</string>
  </property>
  <property name="windowIcon">
   <iconset resource="ldfeditor.qrc">
    <normaloff>:/LDFEditorIcons/Resources/icons/Frame.png</normaloff>:/LDFEditorIcons/Resources/icons/Frame.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>General Properties</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="1">
         <widget class="QLineEdit" name="editFrameName">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="lableName">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>135</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Name*</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Associated Frames</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="LDFCommonTableWidget" name="tableFrames">
          <column>
           <property name="text">
            <string>Frame Name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>ID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Length (Byte(s))</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Publishers</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Subscribers</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonOk">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LDFCommonTableWidget</class>
   <extends>QTableWidget</extends>
   <header>ldfcommontablewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="ldfeditor.qrc"/>
 </resources>
 <connections/>
</ui>
